R/create.R

```diff
@@ -1,23 +1,31 @@
 #' Create a quarto project
 #'
-#' This function calls `quarto create project <type> <name>`. It will create a
-#' new directory with the project name and add some skeletons files for the
-#' project type chosen.
+#' This function calls `quarto create project <type> <name>`. It creates a new
+#' directory with the project name, inside the requested parent directory, and
+#' adds some starter files that are appropriate to the project type.
 #'
 #' # Quarto version required
 #'
-#' This function require Quarto 1.4 or higher. Use [`quarto_version()`]to check
-#' the version of Quarto detected.
+#' This function requires Quarto 1.4 or higher. Use [quarto_version()] to see
+#' your current Quarto version.
 #'
-#' @param type The type of project to create. As of 1.4, it can be one of
+#' @param type The type of project to create. As of Quarto 1.4, it can be one of
 #'   `r paste0("\\code{", paste(quarto_project_type, collapse = "}, \\code{"),"}")`.
 #' @param name The name of the project and the directory that will be created.
-#' @param dir  The directory where to create the new Quarto project.
+#' @param dir The directory in which to create the new Quarto project, i.e. the
+#'   parent directory.
 #'
 #' @seealso Quarto documentation on [Quarto projects](https://quarto.org/docs/projects/quarto-projects.html)
 #'
 #' @inheritParams quarto_render
-#' @inheritParams quarto_add_extension
+#' @param no_prompt Do not prompt to approve the creation of the new project
+#'   folder.
```

Comment:
If the docs for `no_prompt` are inherited from `quarto_add_extension()` they don't really make sense here.

(In `quarto_add_extension()`, `no_prompt` means "Do not prompt to confirm approval to download external extension.")